README.md

```diff
@@ -1,2 +1,58 @@
 # backend
 The backend code serving the VictoryVault API.
+
+# Project structure
+`\alembic`: alembic migrations for automated database management
+
+`\app\__init__.py`: defines init_app() used to start application with uvicorn
+
+`\app\core\config.py`: configure environment variables for database connections
+
+`\app\core\database.py`: define async database manager and helper function to provide async db sessions 
+
+`\app\crud\user.py`: sqlalchemy database CRUD (Create, Read, Update, Delete) operations relating to user model
+
+`\app\models\user.py`: SQLModels used by Pydantic and SQLAlchemy relating to user models
+
+`\app\models\util.py`: generic SQLModels used by Pydantic and SQLAlchemy
+
+
+# Environment setup (TODO: update)
+`\requirements` contains two separate sets of pip requirements, those used during development, and those required to solely run the webapp
+
+The following environment variables must be set:
+```
+DB_USER
+DB_PASSWORD
+DB_HOST
+DB_PORT
+DB_NAME
+```
+
+A PostgreSQL database is expected to be accessible locally (either directly installed and running, or through docker)
```

Comment:
You don't mean "locally" given you've got a DB_HOST variable.

Note that, once we support connecting to the cloud DB, this will need to change - there won't be USER/PASSWORD env vars. (And it's worth recording that here now IMO)